src/Ui.GraphiQL/GraphiQLOptions.cs

```diff
@@ -35,5 +35,16 @@ public class GraphiQLOptions
         /// Gets or sets a delegate that is called after all transformations of the GraphiQL UI page.
         /// </summary>
         public Func<GraphiQLOptions, string, string> PostConfigure { get; set; } = (options, result) => result;
+
+        /// <summary>
+        /// Enables the header editor when <c>true</c>.
+        /// Not supported when ExplorerExtensionEnabled is <c>true</c>.
```

Comment:
Why so ?

Reply:
When `ExplorerExtensionsEnabled` is true, GraphiQL with extensions is registered. This is the current existing behavior, which is maintained for backward compatibility.
Unfortunately, GraphiQL with extensions does not support the header editor.
So to enable it, you have to disable GraphiQL with extensions.

Reply:
OK